src/templates/COLORED_FOLDER.eta

```diff
@@ -0,0 +1,285 @@
+    border-radius: var(--<%~ root %>-border-radius) var(--<%~ root %>-border-radius) 0px 0px;
+}
+
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder.mod-root
+    > .nav-folder-children
+    > .nav-folder.is-collapsed
+    > .nav-folder-title[data-path^="<%~ folder.target %>"] {
+    border-radius: var(--<%~ root %>-border-radius);
+}
+
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder.mod-root
+    > .nav-folder-children
+    > .nav-folder
+    > .nav-folder-title[data-path^="<%~ folder.target %>"] + .nav-folder-children {
+    border-radius: 0px 0px var(--<%~ root %>-border-radius) var(--<%~ root %>-border-radius);
+}
+
+/** Increase Nav File Spacing for Hover Effect to look good */
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder-title[data-path^="<%~ folder.target %>"] .nav-file {
+    margin-left: 0;
+    padding: 0 4px;
+}
+
+/** Remove some padding from file titles to align icons */
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder-title[data-path^="<%~ folder.target %>"] .nav-file-title {
+    padding-left: 8px;
+}
+
+
+/** Center the Folder and Collapse Icons */
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder-title[data-path^="<%~ folder.target %>"] .nav-file-title::before,
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder-title[data-path^="<%~ folder.target %>"] .nav-folder-title-content::before {
+    display: flex;
+    align-self: center;
+    justify-self: center;
+}
+
+/** Margin Control */
+/** This is to override Minimal */
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder-title[data-path^="<%~ folder.target %>"], .workspace-leaf-content[data-type="file-explorer"] .nav-folder-title[data-path^="<%~ folder.target %>"] + .nav-folder-children {
+    margin-left: 6px;
+    margin-right: 6px;
+    border-left: 0;
+    width: auto;
+    min-width: unset; /* Blue Topaz */
+}
+
+<%
+} //end folderSettings
+%>
+
+<% if (it.cfg.hideScrollbar) { %>
+
+/** Scrollbars eliminated */
+.workspace-leaf-content[data-type="file-explorer"] ::-webkit-scrollbar {
+    width: 0px;
+    height: 0px;
+}
+
+<% } %>
+
+
+
+/** Grid */
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder.mod-root > .nav-folder-children {
+    display: grid;
+    gap: 7px;
+    grid-auto-rows: max-content;
+    grid-auto-columns: 100%;
+}
+
+.workspace-leaf-content[data-type="file-explorer"] .nav-folder.mod-root
+    > .nav-folder-children
+    > .nav-folder
+    > .nav-folder-title
+    + .nav-folder-children
+    > .nav-file:last-child {
+    border-right: 1px solid var(--<%~ root %>-folder-background);
+    border-left: 1px solid var(--<%~ root %>-folder-background);
+    border-bottom: 1px solid var(--<%~ root %>-folder-background);
+    border-bottom-left-radius: var(--<%~ root %>-border-radius);
+    border-bottom-right-radius: var(--<%~ root %>-border-radius);
+    padding-bottom: 1px;
+}
+
+/** Relationship lines */
```

Comment:
Should there be an option to enable/disable relationship lines?
(some themes add/color relationship lines, and those settings should work with this snippet, shouldn't they?)